README.md

```diff
@@ -1 +1,72 @@
-# cytoscape.js-matter
\ No newline at end of file
+cytoscape-matterjs
+================================================================================
+
+
+## Description
+
+
+
+
+## Dependencies
+
+ * Cytoscape.js ^x.y.z
+ * <List your dependencies here please>
+
+
+## Usage instructions
+
+Download the library:
+ * via npm: `npm install cytoscape-matterjs`,
+ * via bower: `bower install cytoscape-matterjs`, or
+ * via direct download in the repository (probably from a tag).
+
+`require()` the library as appropriate for your project:
+
+CommonJS:
+```js
+var cytoscape = require('cytoscape');
+var matterjs = require('cytoscape-matterjs');
+
+matterjs( cytoscape ); // register extension
+```
+
+AMD:
+```js
+require(['cytoscape', 'cytoscape-matterjs'], function( cytoscape, matterjs ){
```

Comment:
This call would have to include both `cymatter` and `matter`, as the `matter` dep can't be automatic with AMD